pl_bolts/utils/semi_supervised.py

```diff
@@ -51,19 +57,19 @@ def balance_classes(X: np.ndarray, Y: list, batch_size: int):
     nb_batches = math.ceil(len(Y) / batch_size)
 
     # sort by classes
-    final_batches_x = [[] for i in range(nb_batches)]
-    final_batches_y = [[] for i in range(nb_batches)]
+    final_batches_x: List[Any] = [[] for i in range(nb_batches)]
+    final_batches_y: List[Any] = [[] for i in range(nb_batches)]
```

Comment:
Since these variables are the output of the function, I think they should match with the return type of the function `balance_classes()`:
```python
) -> Tuple[np.ndarray, np.ndarray]:
```
but in order to do that, you might need to change/define other variables, too. We can tighten these types later in another PR, but it'd be nice to do it in this PR. Do you think you can improve typing here?

Reply:
Sure, I will see what I can do. :)